library/Notifications/Widget/ItemList/IncidentHistoryListItem.php

```diff
@@ -31,38 +28,34 @@ class IncidentHistoryListItem extends BaseListItem
     protected function assembleVisual(BaseHtmlElement $visual): void
     {
         $incidentIcon = $this->getIncidentEventIcon();
-        if ($this->item->type === 'incident_severity_changed' || $this->item->type === 'opened') {
-            $content = new Icon($incidentIcon, ['class' => 'severity-' . $this->item->new_severity]);
-        } elseif ($this->item->type === 'rule_matched') {
-            $content = new IcingaIcon($incidentIcon, ['class' => 'type-' . $this->item->type]);
+        if ($incidentIcon === '') {
+            $content = new IconBall('ellipsis', 'fa-solid');
         } else {
-            $content = new Icon($incidentIcon, ['class' => 'type-' . $this->item->type]);
-
             switch ($this->item->type) {
+                case 'incident_severity_changed':
+                    $content = new Icon($incidentIcon, ['class' => 'severity-' . $this->item->new_severity]);
+
+                    break;
+                case 'rule_matched':
+                case 'opened':
                 case 'closed':
-                case 'recipient_role_changed':
+                    $content = new IconBall($incidentIcon);
+
+                    break;
                 case 'notified':
-                    $content->setStyle('fa-regular');
+                case 'escalation_triggered':
+                case 'recipient_role_changed':
+                    $content = new IconBall($incidentIcon, 'fa-solid');
+
+                    break;
+                default:
+                    $content = new IconBall('ellipsis', 'fa-solid');
```

Comment:
With the empty string case gone, this should be possible: (Or an `if..else` instead of a `switch`, your choice)

```suggestion
                default:
                    $content = new IconBall($incidentIcon, 'fa-solid');
```

Reply:
I could also think of using `fa-solid` as default style for `IconBall` :thinking:

Reply:
Is that our default? It makes sense to change it then